### PR TITLE
[Bug] fix insert load should have the same behavior as stream load

### PR DESCRIPTION
**Pull request by spaces-X**
## Proposed changes

see #7179

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 1]
```diff
@@ -1238,6 +1238,13 @@ private void handleInsertStmt() throws Exception {
                     }
                 }
 
+                if (filteredRows > 0 &&
+                        (1 / (1 + (double) loadedRows / filteredRows)) > Config.default_max_filter_ratio) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:1242]: I didn't even know that we have this config....
And I find that it only be used in the old load job, which is deprecated...

So I think better not use it. May be we can add a new session variable for insert operation?

Comment (spaces-X) [reply fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java]: Done

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -324,6 +326,9 @@
     @VariableMgr.VarAttr(name = LOAD_MEM_LIMIT)
     public long loadMemLimit = 0L;
 
+    @VariableMgr.VarAttr(name = INSERT_MAX_FILTER_RATIO)
+    public double insertMaxFilterRatio = 0L;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:330]: The default should be 1.0, to be same as origin.
and please update the document:
```
docs/zh-CN/administrator-guide/variables.md
docs/en/administrator-guide/variables.md
```

Comment (spaces-X) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java]: Done

**Pull request closed without merging**